tests_notebooks/test_manage_app.py

```diff
@@ -0,0 +1,10 @@
+from selenium.webdriver.common.by import By
+
+
+def test_single_app(selenium_driver, final_screenshot):
+    url_params = {"app": "aiidalab-widgets-base"}
+    selenium = selenium_driver("single_app.ipynb", url_params)
+    selenium.set_window_size(1440, 828)
+    selenium.set_window_size(950, 1000)
```

Comment:
Why is it needed to set window size two times?

Reply:
Whoops, just an ommission on my part, fixed.